### DISCUSSION TITLE
Shiny UI hidden behind plot in dashboard

### DISCUSSION THREAD

**Discussion by nclJoshCowley** [quarto-dev/quarto-cli#8802] [Q&A] [answered]
### Description

Hi all, I've just started trying the new dashboard feature from 1.4 and I love it so far, thanks to anyone working on this.

I can't tell if the reprex shared below is an issue or if the fault lies with me.

Essentially, adding a shiny UI with a dropdown is being shown behind the plot.

````qmd
---
format: dashboard
server: shiny
---

## Data {.tabset}

```{r}
#| content: card-toolbar
wide_choices <- letters
names(wide_choices) <- sapply(letters, function(.x) toString(rep(.x, 50)))

shiny::selectizeInput(
  inputId = "par",
  label = "Parameter",
  choices = wide_choices
)
```

```{r}
#| title: "Histogram"
shiny::plotOutput("tmp")
```

```{r}
#| context: server
output$tmp <- 
  shiny::renderPlot(hist(rnorm(1000), xlab = input$par))
```
````

On Quarto version v1.4.550, this is the output:
![image](https://github.com/quarto-dev/quarto-cli/assets/57465489/99d26976-f13b-4abb-af4d-731db6185f02)

**Comment by cderv** [comment, marked as answer]
I believe this has been fixed in Quarto 1.5. If you install the pre-release version this should work. 

It will probably be available in next 1.4 patch  release

Reported at:
* https://github.com/quarto-dev/quarto-cli/issues/8795
